src/Entity/User.php

```diff
@@ -21,11 +24,14 @@ class User
      * @ORM\Id()
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
+     * @Serializer\Expose()
+     * @Groups({"ActivityDetails"})
```

Comment:
Why have you made only this field available in the ActivityDetails? What about the others?